cv32/tests/programs/custom/hello-world/test.yaml

```diff
@@ -2,4 +2,5 @@ name: hello-world
 uvm_test: uvmt_cv32_firmware_test_c
 description: >
     Simple hello-world sanity test
-
+compile_flags:
```

Comment:
Was this an inadvertent checkin?  hello-world should be default config.

Reply:
Whoops!  Thanks, that was _not_ intended...